internal/monitors/collectd/df/metadata.yaml

```diff
@@ -20,26 +20,32 @@ monitors:
       description: Number of inodes that are free.
       included: false
       type: gauge
+      group: reportInodes
     df_inodes.reserved:
       description: Number of inodes reserved for the super user.
       included: false
       type: gauge
+      group: reportInodes
```

Comment:
Just call these groups `inodes` and `percentage`.  There is no need to perpetuate the collectd config field names.